packages/x-date-pickers-pro/package.json

```diff
@@ -47,7 +47,7 @@
     "@date-io/luxon": "^2.15.0",
     "@date-io/moment": "^2.15.0",
     "@mui/utils": "^5.9.3",
-    "@mui/x-date-pickers": "5.0.0",
+    "@mui/x-date-pickers": "5.0.1",
     "@mui/x-license-pro": "5.17.0",
```

Comment:
We should align this dependency version on all packages. 🤔

Reply:
https://github.com/mui/mui-x/pull/6086#discussion_r965988533

I was discussing it here
Theoretically I agree, but I was wondering if for just 1 patch we could avoid adding a weird version bump to our users

Reply:
I'm pretty sure that with previous releases we always bumped `license` package version to the same one as the whole package in all the sub-packages. 🤔 
Correct me if I'm wrong..?

Reply:
Not if you use `year release:version` at least
There is no 5.17.1 for instance https://www.npmjs.com/package/@mui/x-license-pro

Reply:
@LukasTy you answered in the wrong thread I think

Reply:
I'm pretty sure I answered in the correct thread.
Anyways, I'm all for releasing packages once it makes sense. 😉 
Was initially thinking that some other package had a difference license package version, but if they all match - fine by me. 👌 👍